src/core/tests/pass/serialization/custom_ops.cpp

```diff
@@ -60,7 +60,7 @@ TEST_F(CustomOpsSerializationTest, CustomOpNoExtensions) {
             </output>
         </layer>
         <layer name="operation" id="1" type="Template" version="custom_opset">
-            <data  add="11"/>
+            <data num_bodies="0" add="11"/>
```

Comment:
Why did you update old IR?

Reply:
Since I added attribute `num_bodies` to `FrameworkNode` it started to appear in the node. I had to change the IR because otherwise the model comparison in the end of test will fail. I don't think this is too critical, because the IR can be read without issues, maybe there is a better way to solve this issue, not sure.